url.bs

```diff
@@ -1877,8 +1897,9 @@ string <var>input</var>, optionally with a <a>base URL</a> <var>base</var>, opti
 
       <ol>
        <li>
-        <p>If <var>base</var> is non-null and <var>base</var>'s <a for=url>scheme</a> is
-        "<code>file</code>", then:
+        <p>If <var>base</var> is non-null, <var>base</var>'s <a for=url>scheme</a> is
+        "<code>file</code>", and the substring from <var>pointer</var> in <var>input</var> does not
+        <a>start with a Windows drive letter</a>, then:
```

Comment:
Same as above.

Reply:
> What do we gain by removing reference to remaining?

The simpler check, that there are at least two code points:
- without remaining: `substr.length >= 2`
- with remaining: `c != EOF && remaining.length >= 1`